Source/RapyutaSimulationPlugins/Public/Core/RRCrowdAIController.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file RRCrowdAIController.h
+ * @brief Base Crowd AI Controller for Character-based classes, utilizing UCrowdFollowingComponent
+ * @copyright Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+ */
+
+#pragma once
+
+#include "AIController.h"
+
+#include "RRCrowdAIController.generated.h"
+
+UCLASS()
+class RAPYUTASIMULATIONPLUGINS_API ARRCrowdAIController : public AAIController
```

Comment:
Is the class `ARRCrowdAIController` used somewhere?

Reply:
it is better to have separate pr to avoid confusion from next time.